packages/store/src/ivy/ivy-enabled-in-dev-mode.ts

```diff
@@ -21,9 +22,7 @@ export function setIvyEnabledInDevMode(): void {
   try {
     // `try-catch` will also handle server-side rendering, as
     // `window is not defined` will not be thrown.
-    const ng = (window as any).ng;
-    const _ivyEnabledInDevMode =
-      !!ng && typeof ng.getComponent === 'function' && typeof ng.markDirty === 'function';
+    const _ivyEnabledInDevMode = !!(window as any).ng && Number(VERSION.major) >= 9;
```

Comment:
It can be the 9th version, but Ivy is still possible to disable and use the old view engine. You have to remove `markDirty` from the condition.

Reply:
done